src/template/render.rs

```diff
@@ -501,26 +507,68 @@ impl<'a> ChainTemplateSource<'a> {
         let [program, args @ ..] = command.as_slice() else {
             return Err(ChainError::CommandMissing);
         };
-        debug_span!("Executing command", ?command)
-            .in_scope(|| async {
-                let output = Command::new(program)
-                    .args(args)
-                    .output()
-                    .await
-                    .map_err(|error| ChainError::Command {
-                        command: command.to_owned(),
-                        error,
-                    })
-                    .traced()?;
-
-                debug!(
-                    stdout = %String::from_utf8_lossy(&output.stdout),
-                    stderr = %String::from_utf8_lossy(&output.stderr),
-                    "Command success"
-                );
-                Ok(output.stdout)
+
+        let _ = debug_span!("Executing command", ?command).entered();
+
+        // Render the stdin template, if present
+        let input = if let Some(template) = stdin {
+            template
+                .render_stitched(context)
+                .await
+                .map_err(|error| ChainError::Nested {
+                    field: "stdin".into(),
+                    error: error.into(),
+                })
+                .ok()
```

Comment:
Should fail if there's an error here, instead of throwing it away
```suggestion
            let stdin = template
                .render_stitched(context)
                .await
                .map_err(|error| ChainError::Nested {
                    field: "stdin".into(),
                    error: error.into(),
                })?;
            Some(stdin)
```